data/features/m/m.sh

```diff
@@ -17,11 +17,12 @@ m() {
 
   if [ $# -eq 1 ]; then
     man "$1"
-  else
-    for arg in "${@:2}"; do
-      # search for lines that start with the second argument ($2) preceded by one or more spaces (^ +)
-      # The -A5 option tells grep to also output five lines of context after each match.
-      man "$1" | grep -hnI -E "+${arg}" -B 5 -A 5 --color='auto'
-    done
+    return 0
   fi
+
+  for arg in "${@:2}"; do
+    # Search the manual page for the command specified by the first argument ($1)
+    # using the 'man' command, and pipe the output to 'grep' for searching the arguments of the command.
+    man "$1" | grep -hnI -E "+${arg}" -B 5 -A 5 --color='auto'
```

Comment:
Delete the `+`.

If you say that it does not work give me the proof that the output is different with and without `+`

Reply:
https://github.com/AleixMT/Linux-Auto-Customizer/pull/115#discussion_r1168551832